vispy/scene/canvas.py

```diff
@@ -87,17 +87,20 @@ def __init__(self, *args, **kwargs):
         self._fb_stack = []  # for storing information about framebuffers used
         self._vp_stack = []  # for storing information about viewports used
         self._scene = None
-        self._bgcolor = Color(kwargs.pop('bgcolor', 'black')).rgba
```

Comment:
can't it just be `self.bgcolor = kwargs.pop('bgcolor', 'k')` to keep it as one line?